### PR TITLE
Initial draft for autodeploy code

### PR DESCRIPTION
**Pull request by Katsie011**

### CODE REVIEW

**Review by jonoirwinrsa — APPROVED** [review]
lekker

File: cerebrium/fine-tuning/auto-deploying.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Auto-Deploying your Trained Model
+description: Learn how to seamlessly deploy your trained model to a production environment.
+---
+
+# Introduction
+
+If you are reading this guide, chances are that you have successfully trained a model on Cerebrium. Congratulations!
```
Comment (milo157) [inline cerebrium/fine-tuning/auto-deploying.mdx:8]: Remove this line - Developers like getting straight to the point

File: cerebrium/fine-tuning/auto-deploying.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Now that your model has been successfully trained, you have two options for deploying to a production environment:
+
+- The first is to download the trained adapter weights using `cerebrium download-model` and write your own deployment code.
+- The second is to use the built-in auto-deployment feature of Cerebrium's trainer.
+
+While you have more flexibility writing custom deployment code for your finetuned weights, the second option is the method that we recommend for most users!  
+It's faster, easier and gets your model up and running in no time.
```
Comment (milo157) [inline cerebrium/fine-tuning/auto-deploying.mdx:16]: While you have more flexibility writing custom deployment code for your finetuned weights, the second option is faster, easier and gets your model up and running in no time.

File: cerebrium/fine-tuning/auto-deploying.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Auto-deployment streamlines the process of getting your model into a production environment. With this approach, deployment occurs automatically as soon as training is completed. This feature not only saves time but also provides a seamless way to test your model's performance before considering any modifications to the deployment code.
```
Comment (milo157) [inline cerebrium/fine-tuning/auto-deploying.mdx:18]: You can get right of this line. You have mentioned 3 times how seamless and fast autodeployment is - no fluf

File: cerebrium/fine-tuning/auto-deploying.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+The following parameters are the parameters we suggest you supply for an auto-deployment:
+
+```yaml
+autodeploy:
+  name: your-auto-deployed-model-name
+  hardware: AMPERE_A5000 # Hardware to use for the deployment
+  cpu: 2 # Number of CPUs to allocate to the deployment
+  memory: 14.5 # Memory to allocate to the deployment. This depends on your model but for most 7B models, 14.5GB is sufficient.
+  autodeploy-requirements-file: /Path/To/Your/requirements.txt # Requirements that you would need in a cortex deployment
```
Comment (milo157) [inline cerebrium/fine-tuning/auto-deploying.mdx:36]: I don't understand why these are included if we set the main.py? They wont know what libraries we have included

Comment (Katsie011) [reply cerebrium/fine-tuning/auto-deploying.mdx:36]: Imports are generated from the requirements.txt 
It's more there if a user wants a specific version of a package. 

But yes, at the moment they won't know what libraries we've included.

**Pull request merged**